### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by juliantap**
Updated the CCLA process per Tim. We also need to remove the CCLA PDF.

## Change description
Need to update the CCLA process per Tim. 

## Project tracking
n/a

## Do any added TODOs have an issue in the backlog?
no

## Did you add unit tests? Why not?
n/a

## Does this code need functional testing?
no

## Is a code review walkthrough needed? why or why not?
no

## Should additional test coverage be executed in addition to pre-merge?
no

## Does this code need a note in the changelog?
Maybe a note that CCLA process was updated. 

## Does this code require documentation changes?
Only if we decide to note the CCLA chance in changlog.md.

## Additional Information
We also need to remove https://github.com/NetApp/trident/blob/master/CONTRIBUTING_CCLA.pdf from the repo.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,9 +10,8 @@ We appreciate that you want to take the time to contribute! Please follow these
...
 5. Please wait for feedback from someone on NetApp's Trident development team before writing any code. We don't really have an SLA for our feedback, but we will do our best to respond in a timely manner (at least to inform if you are on the right track and that you should proceed, or not).
-6. Download NetApp's [Corporate Contributor License Agreement (CCLA)](https://github.com/NetApp/trident/blob/master/CONTRIBUTING_CCLA.pdf). Read, consider your, and/or your company's, agreement to the document, and sign if appropriate. 
-7. As instructed in the CCLA, please send the completed document to <legalip@netapp.com>. 
-8. If you've made it this far, and have written code that solves the issue, achieves the design you provided, and takes into account whatever comments were provided, then feel free to create your pull request.
+6. Sign and submit NetApp's Contributor License Agreement. You must sign and submit the [Corporate Contributor License Agreement (CCLA)](https://netapp.tap.thinksmart.com/prod/Portal/ShowWorkFlow/AnonymousEmbed/3d2f3aa5-9161-4970-997d-e482b0b033fa) in order to contribute.
```
Comment (gnarl) [inline CONTRIBUTING.md:13]: I think we need to add instructions on selecting Trident for "Project Name" and to use "https://github.com/NetApp/trident" for the "Project Website" field.

**Review by gnarl — CHANGES_REQUESTED** [review]
@juliantap, just one change

File: CONTRIBUTING.md (hunk 1 above)
Comment (juliantap) [reply CONTRIBUTING.md:13]: Added.

**Comment by juliantap** [comment]
> 

@gnarl I added the clarification for Name & Website.

**Pull request closed without merging**